tests/pytests/functional/test_payload.py

```diff
@@ -140,4 +140,16 @@ def test_destroy(sreq, echo_server):
     """
     # ensure no exceptions when we go to destroy the sreq, since __del__
     # swallows exceptions, we have to call destroy directly
-    sreq.destroy()
+    assert sreq.send("clear", "foo") == {"enc": "clear", "load": "foo"}
+    try:
+        sreq.destroy()
+    except Exception as exc:  # pylint: disable=broad-except
+        pytest.fail(f"sreq.destroy threw an exception {exc}")
```

Comment:
I wonder if we should or if we even can tell code cove to ignore this line here as where obviously just trying to catch an exception if it ever occurs. Thoughts?

Reply:
good point. it would be good to ignore it in codecov as expected miss. 

however looking at https://community.codecov.com/t/ignore-lines-with-comments-inside-the-files-expected-misses/2661 i don't think it is possible currently.

Reply:
I could restore this just back to the single call, I changed it to be more verbose about the reason for the fail. but either way will work.

Reply:
> I could restore this just back to the single call, I changed it to be more verbose about the reason for the fail. but either way will work.

I would do just that. If we're catching the exception to fail the test, let the exception fail the test.

Reply:
ok, done